user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,461 @@
+.. _admin-capabilities:
+
+Capabilities List
+=================
+
+The *"Capabilities"* make an important part of the access and permission settings defined in the system, as described in 
+the :ref:`Access and Permissions Basics guide <user-guide-user-management-permissions-basic>`. 
```

Comment:
Edit: The *"Capabilities"* are an important part of the system's access and permission settings, as described in the :ref:`Access and Permissions Basics guide <user-guide-user-management-permissions-basic>`.